assets/src/edit-story/utils/getMediaBaseColor.js

```diff
@@ -55,12 +46,23 @@ function setOrCreateImage(src, onLoad) {
     document.body[BASE_COLOR_NODE] = imgNode;
   }
   imgNode.innerHTML = '';
-  const img = document.createElement('img');
-  img.src = src;
+  const img = new Image();
+  img.crossOrigin = type === 'gif' ? undefined : 'Anonymous';
+  img.addEventListener(
+    'load',
+    function () {
+      const thief = new ColorThief();
```

Comment:
Since release is tomorrow, can we keep this to a minimal diff? E.g. ColorThief instantiation per invocation of `getMediaBaseColor` might be a small performance regression.

The only needed change is the crossOrigin ternary.